scm/doc/TechGuide/chap_quick.rst

```diff
@@ -223,61 +200,80 @@ both building and running the SCM.
-``sp`` and ``w3emc`` (``bacio_ROOT``, ``sp_ROOT``, ``w3emc_ROOT``) environment variables to point
-to their installation paths in order to compile the SCM.
-
-The SCM uses only a small part of the UFS *hpc-stack* package and has fewer
-prerequisites (i.e. no ESMF or wgrib2 needed). Users who are not planning to use the
-UFS can install only NetCDF/NetCDF-Fortran manually or using the
-software package manager (apt, yum, brew).
-
-The Python environment must provide the module for the SCM scripts to
-function. Users can test if f90nml is installed using this command in
+that have not been preconfigured, installing ``spack-stack`` (see :ref:`Section %s <spack-stack>`)
+is highly recommended, as it provides all the necessary prerequisite libraries needed for installing the SCM.
+
+The CCPP/SCM team does not support spack-stack, so users with questions or requiring help with spack-stack installation
+should reference the `spack-stack documentation <https://spack-stack.readthedocs.io/en/latest/>`__.
+However, we have provided an example procedure in
+`this GitHub discussion <https://github.com/NCAR/ccpp-scm/discussions/464>`__.
+
+The main downside to spack-stack is that it contains a large number of libraries and utilities used by the whole
+Unified Forecast System and related applications, only a minority of which are required for the SCM. Users may
+install libraries manually if they wish, but they will need to make sure the appropriate environment variables
+are set to the correct values so that the build system can find them, as described in the following chapter.
+
+
+Setting up compilation environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For users on a pre-configured platform, you can load the spack-stack environment via one of the provided modules in ``scm/etc/modules/``.
+For example, users on the NSF NCAR machine Derecho who wish to use Intel compilers can do the following:
+
+::
+
+   cd [path/to/ccpp-scm/]
+   module use scm/etc/modules/
+   module load derecho_intel
+
+Additionally, for users who have installed spack-stack on their own MacOS or Linux machine can use the provided ``macos_clang``
+or ``linux_gnu`` modules. 
+
+.. note::
+
+  The provided modules assume ``clang``/``gfortran`` compilers on MacOS and GNU compilers for Linux.
+  If you are using a different set of compilers, you may need to modify the module file.
+
+If libraries were installed manually, users will need to set some environment variables
+needed for specifying the location of the various prerequisites. Users will need to set variables for the
+compilers (``CC``, ``CXX``, ``FC``), as well as the root directories for the library installs of NetCDF (``NetCDF_ROOT``),
+``bacio`` (``bacio_ROOT``), ``sp`` (``sp_ROOT``), and ``w3emc`` (``w3emc_ROOT``). This is the procedure used in the
+provided Dockerfile in ``ccpp-scm/docker/``, so users can reference that file for guidance on how to install this software
```

Comment:
Ah, I see that you reference the "manual" approach used in the Dockerfile. That's fine. Disregard the comment above re: spack-stack.